src/entities/addLiquidity/types.ts

```diff
@@ -48,28 +48,32 @@ export type AddLiquidityBaseQueryOutput = {
     vaultVersion: 2 | 3;
 };
 
-export type AddLiquidityWeightedQueryOutput = AddLiquidityBaseQueryOutput;
-
 export type AddLiquidityComposableStableQueryOutput =
     AddLiquidityBaseQueryOutput & {
         bptIndex: number;
     };
 
+export type AddLiquidityWeightedQueryOutput = AddLiquidityBaseQueryOutput;
```

Comment:
This feels like it leads to things being quite strange/tricky to follow. For example, immediately below `AddLiquidityQueryOutput` is making a Union of `AddLiquidityWeightedQueryOutput` & `AddLiquidityBaseQueryOutput` which are both the same.

Reply:
Yeah, I wasn't sure about all this either. There are a couple choices made here:
1. Keeping `AddLiquidityWeightedQueryOutput` alias instead of using `AddLiquidityBaseQueryOutput` for weighted pools directly -> more explicit output name from a user perspective
2. Adding `AddLiquidityBaseQueryOutput` to the union - that's for v3 usage actually, because it doesn't have weighted/csp variations

I could remove the alias and keep it simple - does that sound a better approach? It's not as explicit for the weighted case, but it should be ok 🤷‍♂️